sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/FilterEstimation.scala

```diff
@@ -58,6 +58,7 @@ case class FilterEstimation(plan: Filter) extends Logging {
         rowsAfterFilter = filteredRowCount)
     }
     val filteredSizeInBytes: BigInt = getOutputSize(plan.output, filteredRowCount, newColStats)
+      .min(plan.child.stats.sizeInBytes)
```

Comment:
To fix the test error:
<img width="1260" alt="image" src="https://user-images.githubusercontent.com/5399861/178422367-6aa26b63-1135-4741-8fcc-c2a0551c0fd4.png">